showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITOtelMetrics.java

```diff
@@ -141,8 +141,8 @@ private OpenTelemetryMetricsRecorder createOtelMetricsRecorder(
     return new OpenTelemetryMetricsRecorder(openTelemetry, SERVICE_NAME);
   }
 
-  @Before
-  public void setup() throws Exception {
+  @BeforeEach
```

Comment:
Ah this case I remember, it's because the InMemoryMetricRecorder needs to be cleared every run. It was a limitation when I was writing the test so it should be kept this way.